<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoroidTab</class>
 <widget class="QWidget" name="ChoroidTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="VertSplitter_">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="PathEdit_"/>
       </item>
       <item>
        <widget class="QSplitter" name="LeftSplitter_">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeView" name="DirTree_"/>
         <widget class="QTreeView" name="FilesList_"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="ViewFrame_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
